test/basic_algorithm/modifying_seq_ops.cpp

```diff
@@ -558,17 +558,19 @@ TEST_CASE("alg.basic.unique")
     std::vector<int> vec{1, 2, 2, 3, 3, 3, 4, 4};
 
     SECTION("with iterators") {
-        const auto it = rng::unique(vec.begin(), vec.end());
-        REQUIRE(it == vec.begin() + 4);
+        const auto sub_range = rng::unique(vec.begin(), vec.end());
```

Comment:
Not very important, but `sub_range` with an underscore looks a bit strange, I'd be happy with `sub` or `rng` or even `subrange` :)

Reply:
I wanted to avoid calling the variable `subrange`, to avoid confusion with the type. Changed to `sub`.